### ISSUE TITLE
Accessibility doesn't work

### ISSUE THREAD

**Issue by Suncatcher** [NLLAPPS/ACRPhoneHelper#3]
I installed APH from apk, enabled everything, installed ACR from store bur anyway it shows me that it's not working and I checked the recordings and they are silent.
What went wrong?  Some update broke everything

![Screenshot_2023-02-12-00-30-24-07_d8b4665e962109b1b8e9f9cf4f63b9a5](https://user-images.githubusercontent.com/6388034/218300897-df7e4fb8-2f73-4643-a377-d9777c147c2f.jpg)

![Screenshot_2023-02-12-00-30-40-65_2d087eef9347e1c605596a9ff779fb0a](https://user-images.githubusercontent.com/6388034/218300903-26fa7106-b0f0-4aaa-9921-4964a373db15.jpg)
Phone: Oneplus 9, Android 13 (F.18)

**Comment by NLLAPPS** [comment]
Can you pleas open accessibility settings, turn off APH and turn it back on

**Comment by Suncatcher** [comment]
Done and it seems solved the issue. Any way to make it fail-proof and self-troubleshooting?
The call recording got disabled silently and this happened at the moment I was relying on it, so the calls I was hoping to be recorded are forever lost. And those were very important calls for me. It is not the way of work I expect from the app, it needs to self-signal that something goes wrong and the recording feature is not active.

**Comment by NLLAPPS** [comment]
Accessibility service is managed by Android. While it is possible to detect if it was stopped, sometimes due to alterations made by OEM's  (especially due to excessive power management) service can be stopped without notifiying the app.

Try enabling notification and apply settings from https://dontkillmyapp.com/
